pos/app/(main)/info/page.tsx

```
@@ -0,0 +1,148 @@
+              </div>
+              <div 
+                className="prose prose-indigo max-w-none"
+                dangerouslySetInnerHTML={{ __html: sanitizeDescription(selectedTopic.description) }}
+              />
+            </div>
+          </main>
+        </div>
+      </div>
+    </div>
+  )
+}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)